### PR TITLE
T16557

### PR DESCRIPTION
**Pull request by tchx84**
https://phabricator.endlessm.com/T16557

### CODE REVIEW

File: data/preset/blog.yaml [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+                                - 0
+                                - 2
+                        selection:
+                          type: Selection.AllHighlights
+                          slots:
+                            filter: 'Filter.Sets(invert: true)'
+                            order: 'Order.PublishedDate(ascending: false)'
+
+root:
+  type: Controller.Buffet
```
Comment (ptomato) [inline data/preset/blog.yaml]: Consider `Controller.Mesh`? As far as I can tell from looking at the code, the only thing `Controller.Buffet` has that `Controller.Mesh` doesn't, is support for a context bar, which we don't use.

File: data/preset/blog.yaml [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+                  slots:
+                    card: Card.List
+                selection:
+                  type: Selection.Search
+                  id: search-results
+                no-results:
+                  shortdef: 'Layout.Box(orientation: vertical)'
+                  slots:
+                    contents:
+                    - type: ContentGroup.NoResultsMessage
```
Comment (ptomato) [inline data/preset/blog.yaml]: It occurred to me much later after you asked this morning, that you could just stick `ContentGroup.StaticText` here if you wanted to change the text.

File: data/css/blog.scss [hunk 3]
```diff
@@ -0,0 +1,111 @@
+@import "project-setup";
```
Comment (ptomato) [inline data/css/blog.scss]: Anything from this file we can put in the default theme? This contains the things you added to "make it pretty" but if there's anything we can add that makes the default theme prettier by default, then we should do that.

File: data/css/blog.scss [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  background-color: $background-dark-color;
+  -EknLayout_Sidebar-sidebar-width-small: 252px;
+  -EknLayout_Sidebar-sidebar-width-large: 252px;
+
+  .sidebar {
+    .main-menu {
+      margin: 66px 0px 0px 0px;
+    }
+
+    .CardTitle {
```
Comment (ptomato) [inline data/css/blog.scss]: Seems like you should be able to deduplicate `.CardTitle` with some nested rules here.

File: data/preset/blog.yaml (hunk 2 above)
Comment (tchx84) [inline data/preset/blog.yaml]: I feel very silly now, thanks :)

File: data/preset/blog.yaml (hunk 1 above)
Comment (tchx84) [inline data/preset/blog.yaml]: hmm, we do use the navigation card at the bottom for now, that only works with Buffet controller.

File: data/css/blog.scss (hunk 3 above)
Comment (tchx84) [inline data/css/blog.scss]: I suspect we don't even need that file (at all) anymore, ill remove from here for now.

**Comment by tchx84** [comment]
@ptomato fixed your comments plus other improvements from latest designs

File: data/preset/blog.yaml (hunk 1 above)
Comment (ptomato) [inline data/preset/blog.yaml]: I opened an issue to get rid of the cases where one thing is not supported by one controller.

**Comment by tchx84** [comment]
> I noticed that https://phabricator.endlessm.com/T13175 is a real problem when scrolling down all the way in the sorted "feed". We may have to consider this as a higher priority.

@ptomato I can work on that next, if you think we should.

**Comment by tchx84** [comment]
@ptomato addressed all your comments and created an issue for the default theme styling refactors https://phabricator.endlessm.com/T16689.

**Pull request merged**